src/hooks/useBreakpoint.tsx

```diff
@@ -9,19 +9,23 @@ import { BreakpointKey, Breakpoints } from "../Css";
  * if(breakpoints.mdAndDown) {...do something cool}
  */
 
-type BreakpointsType = Record<BreakpointKey, boolean>;
+type BreakpointsType = Partial<Record<BreakpointKey, boolean>>;
+
+function matchMediaBreakpoints(): BreakpointsType {
+  let bps = {} as BreakpointsType;
+  Object.entries(Breakpoints).forEach(([name, bp]) => {
+    bps[name as keyof BreakpointsType] = window.matchMedia(bp.replace("@media ", "")).matches;
+  });
+  return bps;
+}
```

Comment:
**request:** Can we move these types and the `matchMediaBreakpoints` function to the bottom of the file. That way the hook is the first thing users see when they open the hook file. This would also allow the JSDoc to be associated to the hook function